src/Font.elm

```diff
@@ -128,6 +152,44 @@ fontStyleProperty isItalic =
     if isItalic then "italic" else "normal"
 
 
+-- ANDROID
+
+type alias AndroidFont =
+    String
+
+android : List Font -> List AndroidFont
+android fonts =
+    groupByFamily fonts
+        |> Dict.values
+        |> List.map androidFamily
+
+androidFamily : List Font -> AndroidFont
+androidFamily fonts =
+    "<?xml version=\"1.0\" encoding=\"utf-8\"?>"
+        ++ "\n"
+        ++ "<font-family xmlns:app=\"http://schemas.android.com/apk/res-auto\">"
+        ++ "\n"
+        ++ String.join "\n" (List.map androidFont fonts)
+        ++ "\n"
+        ++ "</font-family>"
+        ++ "\n"
+
+androidFont : Font -> String
+androidFont { weight, isItalic, ttf } =
+    "<font"
```

Comment:
Could you add some indentation to this so that the output might look like this: 

```xml
<font-family ...>
    <font ... />
    <font ... />
</font-family>
```

Reply:
If it helps are tabbing is 4 spaces